libcudacxx/include/cuda/__iterator/zip_iterator.h

```diff
@@ -309,6 +483,7 @@ class zip_iterator : public __zv_iter_category_base<_Iterators...>
     }
   };
 
+  //! @brief Compares two @c zip_iterator for equality. Compares the tuple of stored iterators
```

Comment:
```suggestion
  //! @brief Compares two @c zip_iterator for equality by comparing the tuple of stored iterators
```
(many more occurrences)